sutta_publisher/src/sutta_publisher/edition_parsers/epub.py

```diff
@@ -70,26 +70,37 @@ def __make_chapter_index(
             return _links
 
     def __make_chapter(
-        self, html: BeautifulSoup, chapter_number: int, section_name: str = ""
-    ) -> Tuple[epub.EpubHtml, List[epub.Link]]:
+        self, html: BeautifulSoup, chapter_number: int, section_name: str = "", make_index: bool = True
+    ) -> Tuple[epub.EpubHtml, List[epub.Link] | None]:
```

Comment:
Why don't we use `tuple[epub.EpubHtml, list[epub.Link]]` since python3.10 these types are subscriptable